ui/pages/send/send-header/send-header.component.js

```diff
@@ -19,15 +20,19 @@ export default function SendHeader() {
   const stage = useSelector(getSendStage);
   const asset = useSelector(getSendAsset);
   const t = useI18nContext();
-
+  const draftTransactionExists = useSelector(getDraftTransactionExists);
   const onClose = () => {
     dispatch(resetSendState());
     history.push(mostRecentOverviewPage);
   };
 
-  let title = asset.type === ASSET_TYPES.NATIVE ? t('send') : t('sendTokens');
+  let title = asset?.type === ASSET_TYPES.NATIVE ? t('send') : t('sendTokens');
 
-  if (stage === SEND_STAGES.ADD_RECIPIENT || stage === SEND_STAGES.INACTIVE) {
+  if (
+    draftTransactionExists === false ||
+    stage === SEND_STAGES.ADD_RECIPIENT ||
+    stage === SEND_STAGES.INACTIVE
```

Comment:
Since you used `.includes` on line https://github.com/MetaMask/metamask-extension/pull/15259/files#diff-5fd11e9ad27369ac2d1e53fc158bc9fd26efaf0b9224509843ce70d141bd13bfR97, do you want to use it here?

Reply:
Yup, good call out. Done in latest